File-Formats/DocIO/Working-with-Bookmarks.md

```diff
@@ -1997,4 +1997,8 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## See Also
 
-* [How to replace the content between two different existing bookmarks in a word document](https://www.syncfusion.com/kb/13010/how-to-replace-the-content-between-two-different-existing-bookmarks-in-a-word-document)
\ No newline at end of file
+* [How to replace the content between two different existing bookmarks in a word document](https://www.syncfusion.com/kb/13010/how-to-replace-the-content-between-two-different-existing-bookmarks-in-a-word-document)
+* [How do I insert hidden bookmarks to the document](https://www.syncfusion.com/kb/253/how-do-i-insert-hidden-bookmarks-to-the-document)
+* [How do I insert nested bookmarks to the document](https://www.syncfusion.com/kb/254/how-do-i-insert-nested-bookmarks-to-the-document)
```

Comment:
Modify the title as :

How to insert hidden bookmarks to the document
How to insert nested bookmarks to the document
How to get the bookmarks present in the document